packages/terra-progress-bar/src/ProgressBar.scss

```diff
@@ -2,102 +2,106 @@
 @import './variables';
 @import './mixins';
 
-.terra-ProgressBar {
-  background: #d3d4d5;
-  border-radius: 0.5em;
-  display: inline-block;
-  height: map-get($terra-progress-bar-sizes, 'small');
-  overflow: hidden;
-  width: 100%;
-
-  &:focus {
-    outline: none;
-  }
-
-  &[value] {
-    appearance: none;
-    border: 0; // Remove Firefox and Opera border
-
-    // To fix background color to the proper shade of Grey in Safari
-    &::-webkit-progress-bar {
-      background: #d3d4d5;
-      border-radius: 0;
-    }
-
-    &::-webkit-progress-value {
-      background-color: currentColor;
-      @include terra-border-end(1px solid #FFF);
-      @include terra-border-radius-start(0);
-    }
-
-    &::-ms-fill {
-      background-color: currentColor;
-      @include terra-border-end(1px solid #FFF);
-    }
-  }
-
-  &[value='100'],
-  &[value='0'] {
-    &::-webkit-progress-value {
-      @include terra-border-radius-end(0);
-      @include terra-border-end(0 solid #FFF);
-    }
-
-    &::-ms-fill {
-      @include terra-border-end(0 solid #FFF);
-    }
-  }
-}
-
-// Firefox styles must be entirely separate as it doesn't work well with WebKit styles
-@-moz-document url-prefix('') {
-  .terra-ProgressBar {
+/* stylelint-disable selector-class-pattern  */
+:local {
+  .progressbar {
```

Comment:
Minor:


It looks like previously that progress and bar were considered two different words due to the camel casing. Should this keep to the same standard, making it .progress-bar instead of .progressbar?